packages/cmrjs/cmr-utils.js

```diff
@@ -35,7 +35,7 @@ const { getUrl, xmlParseOptions } = require('./utils');
  * @param {string} stack - the deployment stack name
  * @returns {Object} CMR's success response which includes the concept-id
  */
-async function publish(cmrFile, creds, bucket, stack) {
+async function publishXML2CMR(cmrFile, creds, bucket, stack) {
```

Comment:
I guess we only publish ECHO10 XMLs, and not any of the other formats CMR supports. Out of scope for this effort, I'm going to write a ticket.

Reply:
Yea, this might be worth renaming.  And easy to do.

Reply:
done.